avocado/utils/multipath.py

```diff
@@ -114,6 +115,52 @@ def get_paths(wwid):
     return paths
 
 
+def get_multipath_details():
+    """
+    Get multipath details as a dictionary, as given by the command:
+    multipathd show maps json
+
+    :return: Dictionary of multipath output in json format.
+    """
+    mpath_op = process.system_output("multipathd show maps jsons", sudo=True)
```

Comment:
As noticed on v1, this requires pretty recent multipathd, excluding for instance, all RHEL versions out there.  Anyway, further improvements can be put on top of this if users use this on such platforms.